wagtail/admin/staticfiles.py

```diff
@@ -41,11 +41,11 @@ def versioned_static(path):
     Wrapper for Django's static file finder to append a cache-busting query parameter
     that updates on each Wagtail version
     """
-    base_url = static(path)
-
-    # if URL already contains a querystring, don't add our own, to avoid interfering
-    # with existing mechanisms
-    if VERSION_HASH is None or '?' in base_url:
-        return base_url
+    # if URL already contains a querystring or is an absolute path, then don't add the version
+    # querystring to avoid changing URLs which might interfere with existing mechanisms, or
+    # doesn't need cache-busting anyway
+    if VERSION_HASH is None or '?' in path or path.startswith(('http://', 'https://', '/')):
+        return path
```

Comment:
Afraid this logic isn't correct - the error case occurs when `ManifestStaticFilesStorage` is active and DEBUG is False, i.e. we have a static file backend that handles cache-busting itself, so we want `versioned_static` to revert to the standard behaviour of `static`. In this case, this path is taken (because `VERSION_HASH` is None), but skipping the call to `static` is not valid as it will fail to expand local paths such as `wagtailadmin/js/draftail.js` into valid URLs.